static/app/utils/profiling/routes.tsx

```diff
@@ -200,3 +218,60 @@ export function generateProfileFlamechartRouteWithHighlightFrame({
     },
   });
 }
+
+export function generateProfileRouteFromProfileReference({
+  orgSlug,
+  projectSlug,
+  frameName,
+  framePackage,
+  reference,
+  query,
+}: {
+  frameName: string;
+  framePackage: string | undefined;
+  orgSlug: Organization['slug'];
+  projectSlug: Project['slug'] | undefined;
+  reference: Profiling.ProfileReference;
+  query?: Location['query'];
+}): LocationDescriptor | null {
+  if (!projectSlug) return null;
+
+  if (typeof reference === 'string') {
+    return generateProfileFlamechartRouteWithHighlightFrame({
+      orgSlug: orgSlug,
+      projectSlug: projectSlug,
+      profileId: reference,
+      frameName: frameName,
+      framePackage: framePackage,
```

Comment:
Is it a TODO to add this highlight frame to the continuous profiles?

Reply:
No, we should just add it. If its available it should always link. I missed the reference